### PR TITLE
Add UE plot for buildings to cs2

### PR DESCRIPTION
**Pull request by robinhasse**
This adds UE plots to cs2 for buildings where there was only FE. Additionally, FE and UE plots of heating (without non-heating electricity) are now available. To make the plots comparable, I set the colours explicitly [here](https://github.com/pik-piam/mip/pull/49).

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -32,7 +32,8 @@ Authors@R: c(
     person("Björn", "Sörgel", role = "aut"),
     person("Mike", "Spahr", role = "aut"),
     person("Jessica", "Strefler", role = "aut"),
-    person("Philipp", "Verpoort", role = "aut")
+    person("Philipp", "Verpoort", role = "aut"),
+    person("Robin", "Hasse", role = "aut")
```
Comment (fschreyer) [inline DESCRIPTION:36]: Aren't those files created automatically? Can we manually edit them?

File: inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd [hunk 2]
```diff
@@ -1,22 +1,65 @@
...
+  "UE|Buildings|non-Heating|Electricity|Conventional",
+  "UE|Buildings|Heating|Solids",
+  "UE|Buildings|Heating|Liquids",
+  "UE|Buildings|Heating|Gases",
+  "UE|Buildings|Heating|District Heating",
+  "UE|Buildings|Heating|Electricity|Resistance",
+  "UE|Buildings|Heating|Electricity|Heat pump",
+  "UE|Buildings|Heating|Hydrogen")
+showAreaAndBarPlots(data, items, tot, orderVars = "user", scales = "fixed")
+```
```
Comment (fschreyer) [inline inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd:32]: I think UE/cap would also be nice to plot.

**Review by fschreyer — APPROVED** [review]

**Review by orichters — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (robinhasse) [reply DESCRIPTION:36]: Well I think you have to manually give the name here. `buildLibrary` will then automatically update the zenodo json file and others accordingly...

File: inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd (hunk 2 above)
Comment (robinhasse) [reply inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd:32]: Do you really mean each carrier or total (or total heating) UE / cap?

File: DESCRIPTION [hunk 3]
```diff
@@ -32,7 +32,8 @@ Authors@R: c(
     person("Björn", "Sörgel", role = "aut"),
     person("Mike", "Spahr", role = "aut"),
     person("Jessica", "Strefler", role = "aut"),
```
[the remark this answers is not in the document]
Comment (robinhasse) [reply DESCRIPTION]: Oh thanks, you're absolutely right...

**Pull request merged**